<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TDCat</class>
 <widget class="QMainWindow" name="TDCat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1726</width>
    <height>1295</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TDCat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="handleWidth">
         <number>2</number>
        </property>
        <widget class="CustomTreeView" name="tree_viewer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>2000</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="CustomTabWidget" name="table_tab">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>100</horstretch>
              <verstretch>100</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>SimSun</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QTabBar::tab {
	min-width: 120px;
	min-height: 30px;
}</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="tabsClosable">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>对象</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1726</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="new_conn_action"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>60</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="conn_action"/>
   <addaction name="new_query_action"/>
  </widget>
  <action name="new_conn_action">
   <property name="text">
    <string>新建连接</string>
   </property>
  </action>
  <action name="conn_action">
   <property name="icon">
    <iconset>
     <normalon>:/icon/connection.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="new_query_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/query.png</normaloff>:/icon/query.png</iconset>
   </property>
   <property name="text">
    <string>新建查询</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTreeView</class>
   <extends>QTreeView</extends>
   <header>CustomTreeView</header>
  </customwidget>
  <customwidget>
   <class>CustomTabWidget</class>
   <extends>QTabWidget</extends>
   <header>CustomTabWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../img/resources.qrc"/>
 </resources>
 <connections/>
</ui>
